e2e/app/component/select-menu.ts

```diff
@@ -24,17 +26,13 @@ export default class SelectMenu extends Container {
    * @param {string} textValue Partial or full string to click in Select.
    * @param {number} maxAttempts Default try count is 2.
    */
-  async clickMenuItem(textValue: string, maxAttempts: number = 2): Promise<void> {
+  async selectOption(textValue: string, maxAttempts: number = 2): Promise<void> {
```

Comment:
Presumably the base `select` doesn't work well for this type of menu. Why not just override it?

Reply:
correct. couldn't really use `base-menu` `select` function here.

Reply:
Could you not call this function `select` so that it overrides the `BaseMenu` select? Otherwise there's confusion over which one to use

Reply:
Agreed, it is confusing. I changed it after modifying `select` function in `base-menu` class